src/lib/src/tooltip/tooltip.component.scss

```diff
@@ -8,3 +8,9 @@
   display: inline-block;
   // Top level styles should be nested here
 }
+
+.qa-tooltip-dotted-underline {
+  border-bottom: .1em dotted;
+  border-color: inherit;
+  cursor: help;
```

Comment:
We have a method or mixin for setting CSS cursors. It doesn't do much yet, but it does help us enforce specific cursor usage so we should use it here (don't forget to import the SCSS file).

https://github.com/GetTerminus/terminus-ui/blob/master/src/lib/src/scss/docs/cursor.md